Backend/endpoints/util.go

```diff
@@ -41,6 +41,11 @@ func ParseParamsToSchema(r *http.Request, acceptingMethod string, target interfa
 		return http.StatusBadRequest
 	}
 
+	// Parse multipart file, with max size of 10 MB
+	if err = r.ParseMultipartForm(10 << 20); err != nil {
+		return http.StatusBadRequest
```

Comment:
The tests are failing here. HTTP requests without a multipart file attached to them will throw an error when you try and parse them. Perhaps we should have a separate method for parsing a schema with a multipart file